translations/ui-orders/en.json

```diff
@@ -164,7 +164,7 @@
   "errors.forbiddenDeleteSystemValues": "Can not delete system values",
   "errors.forbiddenDeleteUsedValue": "This value can not be deleted",
   "errors.fundCannotBePaid": "One or more fund distributions on this order can not be encumbered, because there is not enough money in <b>{fundCodes}</b>.",
-  "errors.fundLocationRestrictionViolation": "Unable to open PO. Fund ({fundCode}) is not valid with location ({locationCode})",
+  "errors.fundLocationRestrictionViolation": "Unable to open PO. One or more funds for POL number <b>{polNumber}</b> are not valid with the selected location(s)",
```

Comment:
how is it connected to opening order?

Reply:
The same error message. https://github.com/folio-org/ui-orders/pull/1555/files#diff-78be3eac36203c30b2072b3fe2be4c399302149888b622f22189ac28ebd67190R251

Reply:
why do you have the same messages? `errors.openOrder.fundLocationRestrictionViolation` is for order open, how `errors.fundLocationRestrictionViolation` is connected to order open process?

Reply:
Indeed, the error message has been used. I have removed it.